DEVDOCS.md

```diff
@@ -0,0 +1,60 @@
+#Encore Link Developer Documentation
+EncoreLink is a web application currently built with (from back to front):
+
+* [PostgreSQL Database](https://www.postgresql.org/)
+* [Loopback API Framework](https://loopback.io/)
+* [Node.js Express Web Framework](http://expressjs.com/)
+* [Redux State Container](http://redux.js.org/)
+* [React.js UI Library](https://facebook.github.io/react/)
+* [ES6 JavaScript](http://es6-features.org) with [Babel JS compiler](https://babeljs.io/)
```

Comment:
OK, I'll make those changes. I hate Babel, so did not invest a lot of time researching this one. Maybe your link will help me get my rage under control :)